pkg/kubectl/cmd/clusterinfo/clusterinfo_dump_test.go

```diff
@@ -68,3 +79,109 @@ func TestSetupOutputWriterFile(t *testing.T) {
 		t.Errorf("expected: %v, saw: %v", output, data)
 	}
 }
+
+func TestCmdClusterInfoDump(t *testing.T) {
+	tf := cmdtesting.NewTestFactory()
+	defer tf.Cleanup()
+
+	ns := scheme.Codecs
+
+	encodeResp := func(obj runtime.Object, ver runtime.GroupVersioner) io.ReadCloser {
+		info, _ := runtime.SerializerInfoForMediaType(ns.SupportedMediaTypes(), runtime.ContentTypeJSON)
+		encoder := ns.EncoderForVersion(info.Serializer, ver)
+
+		return ioutil.NopCloser(bytes.NewReader([]byte(runtime.EncodeOrDie(encoder, obj))))
+	}
+
+	encodeRespForCoreGV := func(obj runtime.Object) io.ReadCloser {
+		return encodeResp(obj, corev1.SchemeGroupVersion)
+	}
+
+	encodeRespForAppsGV := func(obj runtime.Object) io.ReadCloser {
+		return encodeResp(obj, appsv1.SchemeGroupVersion)
+	}
+
+	tests := map[string]struct {
+		expNsListReq []string
```

Comment:
Use full names for variables, even when they are longer, it's better for readability.

Reply:
`expectedNamespaces` reads so much better, for example.

Reply:
I started as Java developer and I really like the verbose names for variables and methods. But when in our company we started coding in Go then wet started to be as idiomatic as we can. When I'm writing in given language then I'm trying to apply rules from its eco-system as well.  Because of that, I used here the short names e.g. expected -> exp.

source: https://github.com/golang/go/wiki/CodeReviewComments#variable-names

But if you allowing to have the longer names then I am more than happy.